lib/Subscriber/Behavior/Audit/BlameSubscriber.php

```diff
@@ -92,9 +92,14 @@ public function handlePersist(PersistEvent $event)
             return;
         }
 
-        $node = $event->getNode();
         $locale = $event->getLocale();
 
+        if (!$locale) {
+            return;
```

Comment:
/Should this ever happen? Should it not already by the default locale by this point?